Networking/Networking/Remote/OrderStatsRemoteV4.swift

```diff
@@ -45,5 +52,6 @@ private extension OrderStatsRemoteV4 {
         static let after: String    = "after"
         static let before: String   = "before"
         static let quantity: String = "per_page"
+        static let forceRefresh: String = "force_cache_refresh"
```

Comment:
```suggestion
        static let forceRefresh = "force_cache_refresh"
```

Reply:
Same as above, removed all `String` in https://github.com/woocommerce/woocommerce-ios/pull/7622/commits/373269051a4d18c7cd0e8cd192e86c6faf88f9f4